static/js/settings_emoji.js

```diff
@@ -248,9 +252,7 @@ function show_modal() {
         }
 
         const formData = new FormData();
-        for (const [i, file] of Array.prototype.entries.call($("#emoji_file_input")[0].files)) {
-            formData.append("file-" + i, file);
-        }
+        formData.append("file-0", upload_widget.get_cropped_file());
```

Comment:
This seems like it'll prevent uploading multiple images at once in the emoji upload UI -- is that expected?

Reply:
Yes, it is expected that only one image can be uploaded. Even before I added the cropping feature, the upload widget was restricted to only allow one image. I have maintained the same restriction with uppy as well, so it won't have any impact.